realm/realm-library/src/objectServer/java/io/realm/internal/network/OkHttpAuthenticationServer.java

```diff
@@ -108,6 +109,28 @@ public ChangePasswordResponse changePassword(Token adminToken, String userId, St
         }
     }
 
+    @Override
+    public GetUserProfileValuesResponse getUserProfileValues(Token userToken, URL profileValuesUrl) {
+        //TODO revisit once the EndPoint is defined/known
+        try {
+            String requestBody = GetUserProfileValuesRequest.create(userToken).toJson();
```

Comment:
Another "no biggie".  I think it future-proofs code like this to have a `return` statement at the end.  What would you think of pulling the definition of `requestBody` out of the `try` block, assigning it in the try, and returning based on it, outside the `try`, at the end of the method?